bvm/ballerina-core/src/main/java/org/ballerinalang/channels/ChannelConstants.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.ballerinalang.channels;
+
+/**
+ * Constants related to channel implementation.
+ * @since 0.982.0
+ */
+public class ChannelConstants {
+
+    public static final String DB_NAME = "channels";
+    private static final String TABLE_NAME = "messages";
+    public static final String DB_USERNAME = "SA";
+    public static final String DB_PASSWORD = "";
+    public static final String CONF_NAMESPACE = "b7a.channel.db.";
+    public static final String CONF_USERNAME = "username";
+    public static final String CONF_PASSWORD = "password";
+    public static final String CONF_DB_TYPE = "dbType";
+    public static final String CONF_HOST_OR_PATH = "hostOrPath";
+    public static final String CONF_PORT = "port";
+    public static final String CONF_DB_NAME = "dbName";
+    public static final String CONF_DB_OPTIONS = "dbOptions";
+
+    /**
+     * DB queries.
+     */
+    public static final String CREATE = "create table IF NOT EXISTS " + TABLE_NAME + " (msgId int NOT NULL " +
```

Comment:
We might need to think how we provide create database scripts to other database types as well. As an example, auto_increment will not work for derby.

Reply:
That's a DB admin's duty. We do not create tables from program. This is used only for H2 in memory db creation when no db config available.